dags/trigger_k8s_cronjob.py

```diff
@@ -68,16 +82,10 @@ def trigger_k8s_cronjob(cronjob_name, namespace):
 
         # Create a label_selector from the job's UID
         pod_label_selector = "controller-uid=" + controllerUid
-        try:
-            # Wait a bit for the job to be created
-            time.sleep(10)
-            # Get the pod name for the newly created job
-            pods_list = core_v1.list_namespaced_pod(
-                namespace, label_selector=pod_label_selector, timeout_seconds=10)
-            pod_name = pods_list.items[0].metadata.name
-        except ApiException as e:
-            logging.critical(
-                "Exception when calling CoreV1Api->list_namespaced_pod: %s\n" % e)
+
+         # Wait a bit initially for the job to be created
+        time.sleep(10)
+        pod_name = retry_with_backoff(get_pod_name(namespace, pod_label_selector, core_v1))
```

Comment:
you'll want to pass a lambda here, the `get_pod_name(...)` call doesn't return a function